lib/matrix_convert_to_zero.rb

```diff
@@ -6,5 +6,31 @@
 # Time complexity: ?
 # Space complexity: ?
 def matrix_convert_to_zero(matrix)
-  raise NotImplementedError
+  new_matrix = []
+  rows_w_0 = {}
+  cols_w_0 = {}
+  
+  matrix.length.times do |x|
```

Comment:
This definitely works, but you could also use the 0th index of the row and columns to serve the same role as the `rows_w_0` and `cols_w_0` arrays.